utmap-client/src/components/pages/CalendarPage.js

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  MonthView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  TodayButton,
+} from '@devexpress/dx-react-scheduler-material-ui';
 
-function CalendarPage() {
+//Scheduler is the calendar, today, and taskbar components
+function CalendarPage(props) {
     return (
-        <p>Calendar</p>
-    )
+        <Paper>
+            <Scheduler data={props.events}>
```

Comment:
Javascript has something called "object destructuring" which is good to use wherever possible: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

So, instead of using `props.events` multiple times, you can instead declare a variable called `events` using destructuring:

```js
const { events } = props;
```

This doesn't make a huge difference right now but it can in the future when you have large object chains (e.g. `props.events.locations.names` etc).